leanix_admin/tag_group.py

```diff
@@ -0,0 +1,153 @@
+import json
+
+import leanix_admin.file as file
+import leanix_admin.graphql as graphql
+from leanix_admin.action import BackupAction, RestoreAction
+
+
+def find_by_name(needle, haystack):
+    for current in haystack:
+        if needle['name'] == current['name']:
+            return current
+    return None
+
+
+class TagGroupsBase:
+    def __init__(self, http, graphql_url):
+        self.http = http
+        self.graphql_url = graphql_url
+
+    def _fetch_tag_groups(self, erase_id=True):
```

Comment:
I have no idea what this function does, but it certainly looks good 😄  Would you mind adding a little doc string?